lms/static/scripts/frontend_apps/index.js

```diff
@@ -38,19 +39,27 @@ function init() {
   let app;
   switch (config.mode) {
     case 'basic-lti-launch':
-      services.set(
-        ClientRPC,
-        new ClientRPC({
-          authToken: config.api.authToken,
+      {
+        const { authToken } = config.api;
+        const clientRPC = new ClientRPC({
+          authToken,
           allowedOrigins: config.rpcServer.allowedOrigins,
           clientConfig: /** @type {ClientConfig} */ (config.hypothesisClient),
-        })
-      );
-      services.set(
-        GradingService,
-        new GradingService({ authToken: config.api.authToken })
-      );
-      app = <BasicLTILaunchApp />;
+        });
+        const contentInfoFetcher = new ContentInfoFetcher(authToken, clientRPC);
+        const gradingService = new GradingService({
+          authToken: config.api.authToken,
+        });
+        services.set(ClientRPC, clientRPC);
+        services.set(ContentInfoFetcher, contentInfoFetcher);
+        services.set(GradingService, gradingService);
+
+        if (config.contentBanner) {
+          contentInfoFetcher.fetch(config.contentBanner);
+        }
+
+        app = <BasicLTILaunchApp />;
+      }
```

Comment:
We don't currently have tests for this entry point module. Adding these will require some refactoring so I have left this for a separate PR.

Reply:
Tests for the LMS entry point were added in https://github.com/hypothesis/lms/pull/4289 and tests for the changes here have been added in this branch.